### PR TITLE
SITES-2484-[Accessibility][Container][Sample Content] Insufficient color contrast on Background Image and Nested container examples

### LINKED ISSUE #1730: [Accessibility][Container][Sample Content] Insufficient color contrast on Background Image and Nested container examples
**Steps to Reproduce**
1. In Google Chrome, view Background Image and Nested container examples.
2. Test color contrast of text over a dark section of the background image using a color contrast tool.

**Current Behavior**
Text on the background images has various levels of color contrast ratios given the nature of the changing background colors, but many areas fail to meet the 4.5:1 contrast ratio required of 18 point (24 CSS pixels or less) text.

**Expected behavior/code**
Text of 24 CSS pixels or less meets a 4.5:1 contrast ratio with the background; text larger than that meets a 3:1 contrast ratio. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision and/or limited perception of color.

**WCAG SC**
1.4.3 Contrast

**Screenshot**
The Background Image example, with arrows pointing to dark text with very low contrast over dark parts of the background image.

**Possible Solution**
Use background colors with strong contrast to the text color and/or use high contrast text shadows. Prompt authors to use images that will provide adequate contrast.

### PR DESCRIPTION
**Pull request by comanV**
Creates Background Color Title and Background Color Text styles and uses them to improve contrast on Background Image and Nested container examples.
Provides a field description for Container Component -> Edit Dialog -> Background Image option, suggesting to use an image which will provide adequate contrast.
Fixes #1730
Screenshot 2022-03-15 at 12 22 39
Screenshot 2022-03-15 at 12 22 57

### CODE REVIEW

**Review by vladbailescu — APPROVED** [review]

File: content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
                                                 autoStart="{Boolean}false"
                                                 class="cq-droptarget"
                                                 fieldLabel="Background Image"
+                                                fieldDescription="Background image for the container, please use images that will provide adequate contrast"
```
Comment (jckautzmann) [inline content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml:112]: I'd add "... with the text (accessibility)" at the end of the description.

**Comment by jckautzmann** [comment]
This PR improves the accessibility of the page but the look and feel of the page is heavily impacted. I'm not a fan of the yellowish background colour. I'm wondering if we can improve the accessibility without impacting the look and feel. Maybe we could involve a UX expert to get his/her feedback?
cc: @gabrielwalt

**Comment by comanV** [comment]
> This PR improves the accessibility of the page but the look and feel of the page is heavily impacted. I'm not a fan of the yellowish background colour. I'm wondering if we can improve the accessibility without impacting the look and feel. Maybe we could involve a UX expert to get his/her feedback? cc: @gabrielwalt

A solution without impacting the look and feel would be to choose different background images that provide adequate contrast.

File: content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml (hunk 1 above)
Comment (gabrielwalt) [reply content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml]: Looks good to me.

**Comment by gabrielwalt** [comment]
> > This PR improves the accessibility of the page but the look and feel of the page is heavily impacted. I'm not a fan of the yellowish background colour. I'm wondering if we can improve the accessibility without impacting the look and feel. Maybe we could involve a UX expert to get his/her feedback? cc: @gabrielwalt
> 
> A solution without impacting the look and feel would be to choose different background images that provide adequate contrast.

Yes, I'd just use whatever colors provide sufficient contrast for accessibility. Ideally not too ugly, sure, but I wouldn't start a formal UX process for that. If you're unsure just slack somebody, but this is just sample content, so in the end anything that doesn't break accessibility would work.

**Review by jckautzmann — APPROVED** [review]

**Review by vladbailescu — APPROVED** [review]

**Pull request merged**